tests/lib/nested.sh

```diff
@@ -116,6 +117,23 @@ start_nested_core_vm(){
     fi
 }
 
+add_debug_output_on_nested_core_vm(){
+    cat <<EOF > local.conf
+[Unit]
+StartLimitInterval=0
+[Service]
+Environment=SNAPD_DEBUG_HTTP=7 SNAPD_DEBUG=1
+ExecStartPre=/bin/touch /dev/iio:device0
+EOF
+    copy_remote local.conf
+    execute_remote "sudo mkdir -p /etc/systemd/system/snapd.service.d"
+    execute_remote "sudo mv local.conf /etc/systemd/system/snapd.service.d"
+    execute_remote "sudo systemctl daemon-reload"
+    execute_remote "sudo systemctl stop snapd.socket snapd.service"
```

Comment:
Why not restart instead of stop && start?

Reply:
done